.codesandbox/templates/node/.codesandbox/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM node:18
```

Comment:
fixes deployed sandbox node version
our package.json require >16.20 I think so this fixes that
https://github.com/codesandbox/codesandbox-client/issues/1469#issuecomment-1817786803